### DISCUSSION TITLE
Edit CSS buttonStyle date

### DISCUSSION THREAD

**Discussion by Adminbart76** [add2cal/add-to-calendar-button#511] [Q&A] [answered]
Hello,

I want to change the style of the date button. Could someone help me out? Import CSS from a file (atcb-date.css).
Perhaps I need to alter the lightMode="bodyScheme" ? 

Thanks in advance!
Bart Raymakers

**Comment by jekuer** [comment]
Hi @Adminbart76 ,

Please have a look at the "Override styles" section at https://add-to-calendar-button.com/advanced-use#case-1 (you can use the [styleLight and styleDark options](https://add-to-calendar-button.com/configuration#stylelight) here).
Then, check https://github.com/add2cal/add-to-calendar-button/blob/main/assets/css/atcb-date.css for the variables you can use.

You can also check our new PRO offering, which makes styling a lot more easier and the whole thing way more stable: https://add-to-calendar-pro.com 🚀 

All the best,
Jens

**Comment by Adminbart76** [comment]
Hello Jekuer,
In this example only the font is changed, the rest is not.
styleLight="--btn-background: #2f4377; --btn-text: #fff; --font: Georgia, 'Times New Roman', Times, serif;"

The only thing I need is the background to change. I don't need other pro features. So paying for a other background color is a litle crazy ;-)

**Reply by jekuer** [reply, marked as answer]
The PRO version does not offer more things to change. It only makes it easier.
Please check the linked css file and the variables at the very top. Those are the things you can change - including the background colors.

You're welcome.
